presto-product-tests/README.md

```diff
@@ -35,7 +35,16 @@ broken.
     pip install docker-compose
     ```
 
-### OS X (10.8 "Mountain Lion" or newer)
+### OS X using Docker for Mac (macOS 10.10.3 Yosemite or newer) [PREFERED WAY]
```

Comment:
We can remove "macOs" here, since it's redundant, and the other one doesn't have it